bindings/pyroot/cppyy/CPyCppyy/CMakeLists.txt

```diff
@@ -60,13 +60,18 @@ foreach(val RANGE ${how_many_pythons})
     target_link_libraries(${libname} PUBLIC -Wl,-bind_at_load -Wl,-undefined -Wl,dynamic_lookup cppyy_backend${python_under_version_string})
   elseif(NOT MSVC)
     target_link_libraries(${libname} PUBLIC -Wl,--unresolved-symbols=ignore-all cppyy_backend${python_under_version_string})
+  else()
```

Comment:
Just a matter of taste, but I would do:
`if(APPLE)
...
elseif(MSVC)
...
else
...
```
Looks cleaner to me.

Reply:
As you wish (I simply added the `else()`)